packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/meshes/meshPropertyGridComponent.tsx

```diff
@@ -367,7 +367,10 @@ export class MeshPropertyGridComponent extends React.Component<
 
         if (mesh.morphTargetManager) {
             for (let index = 0; index < mesh.morphTargetManager.numTargets; index++) {
-                morphTargets.push(mesh.morphTargetManager.getTarget(index));
+                const target = mesh.morphTargetManager.getTarget(index);
+                if (target.hasPositions) {

```

Comment:
Hi @Popov72, I'm wondering why here we check `target.hasPositions` instead of any other attribute. Does Babylon assume a morph target having any attribute always has position?

Reply:
Yes, if there are no positions for a morph target, we consider it invalid (the shader code always expects morph positions to exist).

I wonder if we shouldn't relax this constraint; I suppose it would be possible to animate normals/uv/... without animating positions...

Reply:
agreed we can relax that to allow more scenarios to work